keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/DebianArtifactSupplier.kt

```diff
@@ -76,23 +78,31 @@ class DebianArtifactSupplier(
   }
 
```

Comment:
I feel like instead of reading from the repository, this is where the calls to outside services should be?

Then, when this is called the artifact details get returned, and then saved in the database for use elsewhere. Then you don't have to have keelrepository dependency here

Reply:
Done!